gnocchi/tests/functional/gabbits/aggregates-with-resources.yaml

```diff
@@ -127,7 +127,7 @@ tests:
         search: "user_id = 'A50F549C-1F1C-4888-A71A-2C5473CCCEC1'"
         operations: "(aggregate mean (metric cpu.util mean))"
       poll:
-        count: 10
+        count: 60
```

Comment:
it seems like this is just hiding potentially another issue. pifpaf is polling for work every second so even 10s should be more than enough.

Reply:
Yeah I was just trying to see if that was the source of the problem, but clearly it's not. Not sure what it is :(